serverless.yml

```diff
@@ -46,11 +46,5 @@ default-env: &default-env
 functions:
     v1UpsertEdgeDictionaryItem:
       handler: api/controllers/v1/dochub.UpsertEdgeDictionaryItem
-      events:
-        - eventBridge:
-            eventBus:  ${self:custom.dochubEventBus}
-            pattern:
-              source:
-                - ${self:custom.docuhubEventSource.${self:provider.stage}}
```

Comment:
What's the reason for deleting this?

Reply:
Accident i was trying something with event bridge looks like a typo , thanks for catching it